ait/dsn/bin/server.py

```diff
@@ -0,0 +1,275 @@
+        self.raf_instance = ait.dsn.sle.RAF()
+        self.raf_connected = False
+        self.raf_bound = False
+        self.raf_started = False
+
+        self.rcf_instance = ait.dsn.sle.RCF()
+        self.rcf_connected = False
+        self.rcf_bound = False
+        self.rcf_started = False
+
+        self.cltu_instance = ait.dsn.sle.CLTU()
+        self.cltu_connected = False
+        self.cltu_bound = False
+        self.cltu_started = False
+
+        self._route()
+
+        self.testing = True
+
+    def _route(self):
+        self.DSN_api.route('/connect/RAF', callback=self.raf_connect_handler)
+        self.DSN_api.route('/bind/RAF', callback=self.raf_bind_handler)
+        self.DSN_api.route('/start/RAF', callback=self.raf_start_handler)
+        self.DSN_api.route('/stop/RAF', callback=self.raf_stop_handler)
+        self.DSN_api.route('/unbind/RAF', callback=self.raf_unbind_handler)
+        self.DSN_api.route('/disconnect/RAF', callback=self.raf_disconnect_handler)
+        self.DSN_api.route('/connect/RCF', callback=self.rcf_connect_handler)
+        self.DSN_api.route('/bind/RCF', callback=self.rcf_bind_handler)
+        self.DSN_api.route('/start/RCF', callback=self.rcf_start_handler)
+        self.DSN_api.route('/stop/RCF', callback=self.rcf_stop_handler)
+        self.DSN_api.route('/unbind/RCF', callback=self.rcf_unbind_handler)
+        self.DSN_api.route('/disconnect/RCF', callback=self.rcf_disconnect_handler)
+        self.DSN_api.route('/connect/CLTU', callback=self.cltu_connect_handler)
+        self.DSN_api.route('/bind/CLTU', callback=self.cltu_bind_handler)
+        self.DSN_api.route('/start/CLTU', callback=self.cltu_start_handler)
+        self.DSN_api.route('/stop/CLTU', callback=self.cltu_stop_handler)
+        self.DSN_api.route('/unbind/CLTU', callback=self.cltu_unbind_handler)
+        self.DSN_api.route('/disconnect/CLTU', callback=self.cltu_disconnect_handler)
+        self.DSN_api.route('/upload/CLTU', callback=self.cltu_upload_handler)
+
+    def raf_connect_handler(self):
+        if (not self.raf_connected):
+            if self.testing:
+                print ("raf connecting\n")
+            self.raf_instance.connect()
+            self.raf_connected = True
+            if self.testing:
+                print ("raf connected\n")
+        else:
+            ait.core.log.error("RAF instance already connected")
+            bottle.response.status = 400
+
+    def rcf_connect_handler(self):
+        if (not self.rcf_connected):
+            self.rcf_instance.connect()
+            self.rcf_connected = True
+        else:
+            ait.core.log.error("RCF instance already connected")
+            bottle.response.status = 400
+
+    def cltu_connect_handler(self):
+        if (not self.rcf_connected):
+            self.rcf_instance.connect()
+            self.rcf_connected = True
+        else:
+            ait.core.log.error("CLTU instance already connected")
+            bottle.response.status = 400
```

Comment:
Why duplicate handlers for each interface's "actions" when they're functionally identical?

A endpoint such as `/bind/<interface>` could handle all of these without any duplication.